src/components/DigitalConverterComponent.cpp

```diff
@@ -0,0 +1,119 @@
+#include <iostream>
+
+#include <hyro/factory/CommandFactory.h>
+#include <hyro/DigitalConverterComponent.h> 
+
+using namespace std::chrono_literals;
+
+namespace hyro
+{
+hyro::Signal message_signal_rec;
+hyro::Signal message_output;
+Thresholding thold;
+std::shared_ptr<HyroLogger> DigitalConverterComponent::s_logger = HyroLoggerManager::CreateLogger("DigitalConverterComponent");
+
+Result
+DigitalConverterComponent::init (const ComponentConfiguration & configuration)
+{
```

Comment:
This component has two parameters amplitude and threshold. Are you initializing these parameters?

The components' parameters must be initialized using the function ` config.parameters.getParameter`.